app/build-js.sh

```diff
@@ -24,7 +24,7 @@ fi
 pushd ../node_modules/astro-sdk
 npm install
 grunt build_astro_client
-cp js/build/astro-client.js $MYPATH/scaffold-www/
```

Comment:
I believe you made that change, but I will check right now and make a note if we didn't :)